Added TI LDC1312 LDC1314 LDC1612 LDC1614

mki8 opened the pull request:
- [x] Provide a URL to a datasheet for the symbol(s) you are contributing
[http://www.ti.com/lit/ds/symlink/ldc1312.pdf](url)

- [x] An example screenshot image is very helpful
![image](https://user-images.githubusercontent.com/46345438/50663372-b9a50480-0fa9-11e9-99f2-a81dcfdab33b.png)

![image](https://user-images.githubusercontent.com/46345438/50663392-c88bb700-0fa9-11e9-8202-97f8d54d8b03.png)

- [-] Ensure that the associated footprints match the [official footprint library](https://github.com/kicad/kicad-footprints)
I created the footprints for the sensors. [https://github.com/KiCad/kicad-footprints/pull/1257](url)

- [x] If there are matching footprint PRs, provide link(s) as appropriate
[https://github.com/KiCad/kicad-footprints/pull/1257](url)

- [x] Check the output of the Travis automated check scripts - fix any errors as required
No errors shown

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/KiCad/kicad-symbols?pullRequest=1366) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/KiCad/kicad-symbols?pullRequest=1366) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/KiCad/kicad-symbols?pullRequest=1366) it.</sub>

myfreescalewebpage commented:
Hi @mki8, thanks for contributing,

A few comments I have during my review:

- [ ] Descriptions never ends with a dot, to be removed
- [ ] Description of LDC1314 do not end with the correct package name format, should end with `, WQFN-16`
- [ ] LDC13xx parts datasheets should be "http://www.ti.com/lit/ds/symlink/ldc1312.pdf"
- [ ] LDC1312 footprint filter should be `WSON*1EP*4x4mm*P0.5mm*`
- [ ] LDC1314 footprint filter should be `WQFN*1EP*4x4mm*P0.5mm*`
- [ ] Length of the pins should be 100 mil (symbols<100pins)
- [ ] SCL is an Input, not an Output
- [ ] INTB is Open Collector, not Output
- [ ] I will put INxA above INxB on the symbol
- [ ] INxx are Inputs, not Outputs
- [ ] LDC1312 PAD number is not correct, it should be 13 instead of 17
- [ ] Globally the position of the pins is not very conventional, something like shown on the simplified schematic on the first page of the datasheet looks better

Above URL of the footprint PR are not correct, this is working: https://github.com/KiCad/kicad-footprints/pull/1257

Cheers,
Joel

mki8 commented:
I reworked the the symbol according the datasheets simplified schematic and added the remarks of Joels review.
![image](https://user-images.githubusercontent.com/46345438/50724102-1ad2f200-10e8-11e9-9594-0349be83bb99.png)
![image](https://user-images.githubusercontent.com/46345438/50724113-41912880-10e8-11e9-997f-4b0fd412dc29.png)

myfreescalewebpage commented:
> I added the dot because of this example in the KiCad Library Convention:

You are right, it is to be updated.

Your symbols looks very nice. A single comment now:

- [ ] You have also modified LDC16xx which were already correct: http://www.ti.com/lit/ds/symlink/ldc1612.pdf

Then we will just need to wait the footprint to be merged.

Joel

poeschlr commented:
New footprint pull request: https://github.com/KiCad/kicad-footprints/pull/1274
